code/components/gta-streaming-five/src/ScaleformHacks.cpp

```diff
@@ -389,6 +401,46 @@ namespace sf
 	}
 }
 
+namespace sf::logging
+{
+	const char* const g_scriptTypes[] = {
+		"GENERIC_TYPE",
+		"SCRIPT_TYPE",
+		"HUD_TYPE",
+		"MINIMAP_TYPE",
+		"WEB_TYPE",
+		"CUTSCENE_TYPE",
+		"PAUSE_TYPE",
+		"STORE",
+		"GAMESTREAM",
+		"SF_BASE_CLASS_VIDEO_EDITOR",
+		"SF_BASE_CLASS_MOUSE",
+		"SF_BASE_CLASS_TEXT_INPUT",
+	};
+
+	static bool g_scaleformDebugLog = 0;
+	static ConVar<bool> g_scaleformDebugLogVar("cl_scaleform_debug_log", ConVar_Archive | ConVar_UserPref, false, &g_scaleformDebugLog);
```

Comment:
I think for game behavior we generally use `game_` prefix, so something like `game_enableScaleformDebugLog`?

Reply:
aight, looks better yeah.